package.json

```diff
@@ -43,7 +43,7 @@
     "prepublishOnly": "yarn clean && yarn build",
     "pretest:base": "cross-env DEBUG=eslint-plugin-svelte*",
     "preversion": "yarn test && git add .",
-    "svelte-kit": "node --experimental-loader ./svelte-kit-import-hook.mjs node_modules/@sveltejs/kit/svelte-kit.js",
+    "svelte-kit": "node --experimental-loader ./svelte-kit-import-hook.mjs node_modules/vite/bin/vite.js --config vite.config.mjs",
```

Comment:
Can we use `yarn ts` here? And `vite.config.ts`(or `.mts`).

Reply:
Besides, a `vite` script with `yarn ts node_modules/vite/bin` should be added, and reuse `yarn vite` here.

Reply:
Thank you for letting me know! I didn't know that vite.config.mts was available.

Reply:
You know, Vite uses `esbuild` under the hood. 🥰

Reply:
Hmm. It didn't work. Seems confusing with ESM and CJS. Probably because tsconfig is configured for CJS.

Reply:
Fine, then maybe we've just found a bug of Vite. 😅

Reply:
It may be resolved in the next release 😄 
https://github.com/vitejs/vite/pull/8729

Reply:
Maybe we can use the beta version?

Reply:
Fair enough. 😅